hack/check-license-header.sh

```diff
@@ -1,17 +1,6 @@
 #!/usr/bin/env bash
-# Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# SPDX-FileCopyrightText: 2019 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
can you explain how you have changed the license in the files? The year is changed wrongly here. I haven't/can't check all the files, just had a luck to find this.
```suggestion
# SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
```

Reply:
I did a search and replace. Maybe I mixed up some stuff for the bash scripts as they have a different commenting style. I will go over everything again and make sure that the years are matching.